source/language/standard_library.rst

```diff
@@ -0,0 +1,329 @@
+   :math:`\lvert1\rangle \to e^{i\lambda}\lvert1\rangle`.
+
+   Equivalent to ``ctrl @ gphase(λ) a``.
+
+   .. seealso::
+
+      :gate:`rz`
+         The same operation up to a global phase.
+
+      :gate:`phase`
+         An alternative name for backwards compatibility with OpenQASM 2.
+
+   .. versionadded:: 3.0
+
+.. gate:: x a
+
+   The Pauli X gate.  Defined by the mapping :math:`\lvert0\rangle \leftrightarrow \lvert1\rangle`.
+
+   .. versionadded:: 3.0
+
+.. gate:: y a
+
+   The Pauli Y gate.  Defined by the mapping :math:`\lvert0\rangle \to i\lvert1\rangle` and
+   :math:`\lvert1\rangle \to -i\lvert0\rangle`.
+
+   .. versionadded:: 3.0
+
+.. gate:: z a
+
+   The Pauli Z gate.  Defined by the mapping :math:`\lvert0\rangle \to \lvert0\rangle` and
+   :math:`\lvert1\rangle \to \lvert1\rangle`.
+
+   Equivalent to ``p(pi) a``.
+
+   .. versionadded:: 3.0
+
+.. gate:: h a
+
+   The Hadamard gate.  Defined by the mapping
+   :math:`\lvert0\rangle \to \bigl(\lvert0\rangle + \lvert1\rangle\bigr)/\sqrt2` and
+   :math:`\lvert1\rangle \to \bigl(\lvert0\rangle - \lvert1\rangle\bigr)/\sqrt2`.
+
+   .. versionadded:: 3.0
+
+.. gate:: s a
+
+   The :math:`\sqrt Z` gate (see :gate:`z`).  The square root is chosen conventionally, that is
+   the gate is equivalent to :math:`P(\pi/2)`, in terms of :gate:`p`.
+
+   .. versionadded:: 3.0
+
+.. gate:: sdg a
+
+   Adjoint of :gate:`s`.  Equivalent to :math:`P(-\pi/2)`, in terms of :gate:`p`.
+
+   .. versionadded:: 3.0
+
+.. gate:: t
+
+   The :math:`\sqrt S` gate (see :gate:`s`).  The square root is chosen conventionally, that is
+   the gate is equivalent to :math:`P(\pi/4)`, in terms of :gate:`p`.
+
+   .. versionadded:: 3.0
+
+.. gate:: tdg a
+
+   Adjoint of :gate:`t`.  Equivalent to :math:`P(-\pi/4)`, in terms of :gate:`p`.
+
+   .. versionadded:: 3.0
+
+.. gate:: sx a
+
+   The :math:`\sqrt X` gate (see :gate:`x`).
+
+   Explicitly, this has the action
+   :math:`\lvert0\rangle \to \Bigl((1 + i)\lvert0\rangle + (1 - i)\lvert1\rangle\Bigr)/2` and
+   :math:`\lvert1\rangle \to \Bigl((1 - i)\lvert0\rangle + (1 + i)\lvert1\rangle\Bigr)/2`.
+
+   .. versionadded:: 3.0
+
+.. gate:: rx(θ) a
+
+   Rotation about the :math:`X` axis: :math:`\exp(-i\theta X)`.
+
+   .. versionadded:: 3.0
+
+.. gate:: ry(θ) a
+
+   Rotation about the :math:`Y` axis: :math:`\exp(-i\theta Y)`.
+
+   .. versionadded:: 3.0
+
+.. gate:: rz(θ) a
+
+   Rotation about the :math:`Z` axis: :math:`\exp(-i\theta Z)`.  Note that this differs from
+   :gate:`p` by a global phase of half the rotation angle.
+
+   .. seealso::
+      :gate:`p`
+         The same gate but with a different global-phase convention.
+
+   .. versionadded:: 3.0
+
+
+Two-qubit gates
+...............
+
+All of the controlled gates defined in the standard library follow the same conventions of the
+``ctrl`` modifier.
+Explicitly, the first qubit is the control and the second the target.
```

Comment:
I was taught, and still follow, using "in particular" rather than "explicitly" here. There are an unspecified number of "same conventions". Using "in particular" (especially if it is understood as a math/science idiom) says that we are drawing attention to one particular item as especially important in the present context.

I might be accused of proscriptivism, but I think it is sometimes warranted.

Reply:
The controlled gates have all the same conventions as the controlled modifier (it just happens that there's only really one), so I think "in particular" would be misleading - we're not selecting one of many, we're linking together and then being explicit about what those linked conventions are.

Reply:
Ok. This is clear.